client-go/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2014 Engine Yard, Inc.
```

Comment:
For a new copyright notice we should use the year the code was introduced: 2015.